presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreOperationStats.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import org.joda.time.DateTime;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class MetastoreOperationStats
+{
+    private final MetastoreOperationType operationType;
+    private final List<DateTime> lastDataCommitTimes;
+    private static final int DUMMY_MILLISECONDS = -1000;
```

Comment:
Why using -1000? If we just need a special value as the default value, it might be good to use 0 in case some downstream pipeline use unsigned int as the type.

Reply:
This number is for DEPA hive table; for unpartitioned tables, or unknown values, they require us to report -1. -1 second ~ -1000 milli. I can find a way to give the value just before the reporting, and we can try to use 0 instead here.